charts/csi-secret-provider-class/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: csi-secret-provider-class
```

Comment:
```suggestion
name: csi-secret-provider-class
home: https://github.com/adfinis-sygroup/helm-charts/tree/master/charts/csi-secret-provider-class
```

Reply:
yepp, i pushed this together with the other things needed for the linter to pass